src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {
+  Dispatch,
+  FC,
+  SetStateAction,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { ErrorType } from '../types/ErrorType';
+
+type Props = {
+  onAddTodo: (value: string) => Promise<void>;
+  setErrorTodos: Dispatch<SetStateAction<ErrorType>>;
+};
+
+export const TodoHeader: FC<Props> = ({ onAddTodo, setErrorTodos }) => {
+  const [inputValue, setInputValue] = useState('');
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
```

Comment:
This effect works only once on the load page.
It does not re-set focus after adding a new todo